src/openapi.json

```diff
@@ -1163,6 +1163,12 @@
 						"type": "string",
 						"format": "date-time",
 						"readOnly": true
+					},
+					"notAfter": {
+						"type": "string",
+						"format": "date-time",
+						"readOnly": true,
+						"nullable": true
```

Comment:
Do we want to expose this in the API? I feel like there's an argument to be made that this is an implementation detail, and it's okay to not return "deleted" permissions.

Reply:
It seemed to me that `notAfter` could also eventually allow for people to set expiring permissions (though we are not opting to support it) and in that lens it could be included as... foreshadowing!

I'm fine not exposing it.  I'm not sure how we would test it without writing a direct SQL query if we aren't exposing it, though maybe that's a-ok to not test (and instead just verify that no value gets returned when you try to select it)!

Leaving it as is is basically partially implementing expiring permission support; making it more obscure is also fine and means we don't have a partial implementation of something; the code wasn't that hard to write and can be easily re-written later.

Reply:
I lean toward leaving it out: simpler, committing to less, yagni, etc.